ci/scripts/run-backfill-tests.sh

```diff
@@ -151,7 +151,7 @@ test_backfill_tombstone() {
     done
   ' 1>deletes.log 2>&1 &
 
-  risedev psql -c "CREATE MATERIALIZED VIEW m1 as select * from tomb;"
+  risedev psql -c "set backfill_rate_limit = 200;set streaming_use_snapshot_backfill = true; CREATE MATERIALIZED VIEW m1 as select * from tomb;"
```

Comment:
Try remove the backfill_rate_limit setting. Otherwise backfill will take a really long time. When it's set, backfill will not scan `at least one row` per epoch.